places.py

```diff
@@ -0,0 +1,143 @@
+from types import resolve_bases
+import requests, json
+import geocoder
+from geopy.geocoders import Nominatim
+import os
+from dotenv import load_dotenv
+
+load_dotenv()
+
+API_KEY = os.getenv("GOOGLE_MAPS_API_KEY")
+
+
+def miles_to_meters(miles):
+    try:
+        return miles * 1_609.344
+    except:
+        return 0
+
+
+# def get_address(latitude, longitude):
+#     geolocator = Nominatim(user_agent="geoapiExercises")
+#     location = geolocator.reverse((latitude, longitude))
+#     address = location.raw["address"]
+#     if "house_number" in address:
+#         house_num = address["house_number"]
+#     else:
+#         house_num = ""
+
+#     return str(
+#         house_num
+#         + " "
+#         + address["road"]
+#         + ", "
+#         + address["city"]
+#         + ", "
+#         + address["state"]
+#         + " "
+#         + address["postcode"]
+#         + ", "
+#         + address["country"]
+#     )
```

Comment:
You did not remove your commented code from the file. Can you please remove them.